data/json/recipes/other/tool.json

```diff
@@ -90,6 +90,28 @@
     ]
   },
   {
+    "result": "pickaxe",
+    "type": "recipe",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "fabrication",
+    "difficulty": 7,
+    "time": 420000,
+    "autolearn": true,
+    "book_learn": [ [ "textbook_carpentry", 5 ], [ "textbook_fabrication", 6 ] ],
+    "using": [ [ "forging_standard", 4 ], [ "steel_standard", 4 ] ],
```

Comment:
That was a direct translation of the 4 lumps of steel into forging_standard, if I recall correctly? How much should it be instead, 2 perhaps?